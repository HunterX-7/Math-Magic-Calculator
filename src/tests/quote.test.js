import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';

it('Quote page renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
