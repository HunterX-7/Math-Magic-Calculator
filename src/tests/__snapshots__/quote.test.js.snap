// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page renders correctly 1`] = `
<div
  className="container p-5"
>
  <h2
    className="quote-margin"
  >
    "Pure mathematics is, in its way, the poetry of logical ideas"
  </h2>
  <p
    className="text-center"
  >
    <strong>
      Albert Einstein, German theoretical physicist
    </strong>
  </p>
</div>
`;
